### DISCUSSION TITLE
STM32F405RGT6 Frozen Code Access File System Files

### DISCUSSION THREAD

**Discussion by AngeloFatica** [micropython/micropython#11395] [STM32 / Pyboard] [answered]
Is there any way for frozen code to access files inside the file system? i.e. Can a file that is running frozen open a text file inside the file system with read and write capabilities? Or are there other ways to be able to r/w these files with frozen code? Thanks.

**Comment by robert-hh** [comment, marked as answer]
Frozen bytecode can do file access like any other Python code. There is nothing special with it.

**Reply by AngeloFatica** [reply]
I've tried accessing files within the filesystem with frozen code and it seemed to error out with multiple permutations. Ive tried opening the file with just the file name as the path and ive also tried opening the file with /flash/filename as well and it doesnt seem to work.

**Reply by robert-hh** [reply]
What were the error messages? Does the filesystem exist? Was it mounted?

**Reply by AngeloFatica** [reply]
I was going to recompile the bytecode to give you the example but I noticed my error when opening the file from the file system... I tried opening "flash/file" instead of "/flash/file". Simple mistake. Thank you for your help in the meantime!
